xt/certs-and-keys/tls.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# Generate self signed root CA cert
+openssl req -days 365 -config my.conf -nodes -x509 -newkey rsa:4096 -keyout ca.key -out ca-crt.pem -subj "/C=CZ/ST=Central Bohemia/L=Prague/O=CA/OU=IT/CN=localhost/emailAddress=foo@example.net"
```

Comment:
Do we not need a usable email address here?

Reply:
Nah, a self-signed cert will never validate with anybody but us. A fake email is okay, imo.